src/common/actionEnums/login.ts

```diff
@@ -0,0 +1,5 @@
+
+export const loginActionEnums = {
+  USERPROFILE_UPDATE_EDITING_LOGIN:  'USERPROFILE_UPDATE_EDITING_LOGIN',
+  USERPROFILE_PERFORM_LOGIN : 'USERPROFILE_PERFORM_LOGIN'
+}
```

Comment:
This will need a further refactor, but right now to avoid collisions let's add a unique prefix (pseudocode):

```
const prefix = 'loginActions_';

export const loginActionEnums {
   USERPROFILE_UPDATE_EDITING_LOGIN:  `{$loginActions}USERPROFILE_UPDATE_EDITING_LOGIN`,
   USERPROFILE_PERFORM_LOGIN : `{$loginActions}USERPROFILE_PERFORM_LOGIN`
}
```

We should open a case to further this case this but just a future refactor discussion